src/index/Vocabulary.cpp

```diff
@@ -231,11 +231,20 @@ void Vocabulary<S, ComparatorType, I>::setLocale(const std::string& language,
       ComparatorType(language, country, ignorePunctuation);
 }
 
+// _____________________________________________________________________________
+template <typename S, typename C, typename I>
+auto Vocabulary<S, C, I>::getBoundsForWord(std::string_view word) const
+    -> std::pair<IndexType, IndexType> {
+  return {IndexType::make(
+              vocabulary_.getPositionOfWordLower(word).indexOrDefault(size())),
+          IndexType::make(
+              vocabulary_.getPositionOfWordUpper(word).indexOrDefault(size()))};
```

Comment:
As discussed, the `getPositionOfWord` function should just return a pair of two indices
(this can then also be used in the `getId` function, if the upperBound is equal to the lower bound, then the word was not found.